stdlib/2/pydoc.pyi

```diff
@@ -16,20 +16,20 @@ __version__ = ...  # type: str
 __credits__ = ...  # type: str
 
 def pathdirs() -> List[str]: ...
-def getdoc(object: object) -> AnyStr: ...
+def getdoc(object: object) -> Union[str, unicode]: ...
 def splitdoc(doc: AnyStr) -> Tuple[AnyStr, AnyStr]: ...
 def classname(object: object, modname: str) -> str: ...
 def isdata(object: object) -> bool: ...
 def replace(text: AnyStr, *pairs: AnyStr) -> AnyStr: ...
 def cram(text: str, maxlen: int) -> str: ...
 def stripid(text: str) -> str: ...
 def allmethods(cl: type) -> MutableMapping[str, MethodType]: ...
-def visiblename(name: str, all: Optional[List[str]] = ..., obj: Optional[object] = ...) -> bool: ...
+def visiblename(name: str, all: Optional[Container[str]] = ..., obj: Optional[object] = ...) -> bool: ...
 def classify_class_attrs(object: object) -> List[Tuple[str, str, type, str]]: ...
 
 def ispackage(path: str) -> bool: ...
-def source_synopsis(file: IO[AnyStr]) -> Optional[str]: ...
-def synopsis(filename: str, cache: MutableMapping[str, Tuple[int, str]] = ...) -> Optional[str]: ...
+def source_synopsis(file: IO[AnyStr]) -> Optional[AnyStr]: ...
+def synopsis(filename: str, cache: MutableMapping[str, Tuple[int, str]] = ...) -> Optional[AnyStr]: ...
```

Comment:
No, for source_synopsis AnyStr is right (unless I'm misreading the code)—if you pass it an `IO[unicode]`, it will give you a `unicode` back.